include/aktualizr-lite/tuf/tuf.h

```diff
@@ -0,0 +1,142 @@
+#ifndef AKLITE_TUF_TUF_H_
+#define AKLITE_TUF_TUF_H_
+
+#include <iostream>
+#include <string>
+
+#include <boost/algorithm/string.hpp>
+#include <boost/filesystem.hpp>
+#include <boost/program_options.hpp>
+#include <boost/program_options/variables_map.hpp>
+#include <boost/property_tree/ptree.hpp>
+
+#include "crypto/keymanager.h"
+#include "http/httpclient.h"
+#include "json/json.h"
+#include "libaktualizr/config.h"
+#include "storage/invstorage.h"
+#include "uptane/fetcher.h"
+#include "uptane/imagerepository.h"
+
+#include "aktualizr-lite/tuf/tuf.h"
+
+namespace Uptane {
+class Target;
+}
+
+namespace aklite {
+namespace tuf {
+
+// enum class RoleEnum { Root = 0, Snapshot = 1, Targets = 2, Timestamp = 3, InvalidRole = -1 };
+// virtual std::string fetchRole(RoleEnum role, int version) = 0;
+
+// TODO: consider switching to a single fetchRole
+class TufRepoSource {
```

Comment:
I think we can remove the `Tuf` prefix from all type names since they are under/in `tuf` namespace.

Reply:
Done